telemetry_api/lib/telemetry_api/operators.ex

```diff
@@ -142,24 +137,6 @@ defmodule TelemetryApi.Operators do
     end
   end
 
-  @doc """
-  Updates a operator.
-
-  ## Examples
-
-      iex> update_operator(operator, %{field: new_value})
-      {:ok, %Operator{}}
-
-      iex> update_operator(operator, %{field: bad_value})
-      {:error, %Ecto.Changeset{}}
-
-  """
-  def update_operator(%Operator{} = operator, attrs) do
-    operator
-    |> Operator.changeset(attrs)
-    |> Repo.update()
-  end
```

Comment:
I'm currently using this one here: https://github.com/yetanotherco/aligned_layer/pull/1186/files#diff-84f700b2608821ebb00e5350ef544a8c4e235cee1e465ba75f071beb9d7101b1R50-R54

We may have conflicts.